sql/core/src/main/scala/org/apache/spark/sql/execution/command/functions.scala

```diff
@@ -60,7 +60,7 @@ case class CreateFunctionCommand(
       // We first load resources and then put the builder in the function registry.
       // Please note that it is allowed to overwrite an existing temp function.
       catalog.loadFunctionResources(resources)
-      catalog.registerFunction(func, ignoreIfExists = false)
+      catalog.registerFunction(func, overrideIfExists = false)
```

Comment:
This looks a bit weird. The above comment says `Please note that it is allowed to overwrite an existing temp function.`. But we don't override it if it exists.